RSC Publishing.js

```diff
@@ -73,18 +73,21 @@ function scrape(doc, type) {
 		if(item.tags.length == 1 && !item.tags[0]) {
 			item.tags = [];
 		}
-		
+
 		if (item.date) {
 			item.date = ZU.strToISO(item.date);
 		}
 
-		if (item.itemType == "bookSection") {
-			if (item.DOI){
-				if (item.extra) item.extra += "\nDOI: " + item.DOI;
-				else item.extra = "DOI: " + item.DOI;
+		if (item.DOI && !ZU.fieldIsValidForType("DOI", item.itemType)){
+			if (item.extra) {
+				if (item.extra.search(/^DOI:/) == -1) {
+					item.extra += "\nDOI: " + DOI;
+				}
+			} else {
+				item.extra = "DOI: " + DOI;
```

Comment:
it's needed because this is recognized by EM as a journal article so it doesn't move the DOI. We change that above.